Basic-Car-Maintenance/Shared/Utilities/Bundle+extension.swift

```diff
@@ -16,4 +16,8 @@ extension Bundle {
     var buildNumber: String {
         return infoDictionary?["CFBundleVersion"] as? String ?? "1"
     }
+    
+    var keychainAccessGroup: String? {
+        return infoDictionary?["KeychainAccessGroup"] as? String
+    }
```

Comment:
should this have a default value if nothing is found?

Reply:
I think it should. I wasn't sure if we should hardcode the team name into the application.

The default value would be "<team-id>.<bundleId>"